spec/inject_remake_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'inject_remake'
+
+describe Array do
+
+  it 'adds up an array of numbers' do
+  expect([1,2,3,4].inject_remake(){|total, element| total + element}).to eq [1,2,3,4].inject{|total, element| total + element}
+  end
+
+  it 'adds up an array of numbers with a given starting point' do
+  expect([1,2,3,4].inject_remake(10){|total, element| total + element}).to eq [1,2,3,4].inject(10){|total, element| total + element}
+  end
+
+  it 'can multiply an array of numbers' do
+  expect([1,2,3,4].inject_remake(){|total, element| total * element}).to eq [1,2,3,4].inject{|total, element| total * element}
```

Comment:
Line is too long. [126/80]<br>Use 2 (not 0) spaces for indentation.<br>Do not use parentheses for method calls with no arguments.<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.